nemo_aligner/models/nlp/gpt/gpt_reward_model.py

```diff
@@ -162,6 +162,7 @@ def __init__(
         share_embeddings_and_output_weights: bool = False,
         position_embedding_type: Literal["learned_absolute", "rope"] = "learned_absolute",
         rotary_percent: float = 1.0,
+        rotary_base: int = 10000,
```

Comment:
Shouldn't this param also be added to the `super().__init__()` call below?

Reply:
ah! thanks for catching it 39904668357a9559e7ca97e6980abee2ab6c12d4